lms/static/js/certificates/models/certificate_exception.js

```diff
@@ -24,6 +24,16 @@
                     notes: ''
                 },
 
+                urlRoot: function(){
+                    return this.get('urlRoot');
+                },
+
+                url: function() {
```

Comment:
This would be obsolete if when we get rid of `certificate_exception_id` in the certificate exception view as suggested above.